backend/core/utils/email_domain.py

```diff
@@ -0,0 +1,49 @@
+"""
+Shortcut functions related to the email domain.
+"""
+import datetime
+
+from django.db.models.query import RawQuerySet
+
+from core.models.user import EmailDomain
+
+
+def get_user_cnt_per_email_domain(
```

Comment:
Small renaming suggestion

`get_email_domain_with_recent_new_users`

Because
(1) The function does not return user counts
(2) The name does not indicate anythion about that the function does filters email domains based on dates.

Reply:
Thanks this name is better, I changed it. But the function return a querry set which contains the number of accounts created for each domain.